concepts/0302-aries-interop-profile/README.md

```diff
@@ -130,102 +130,109 @@ Feature | [0056-service-decorator](https://github.com/hyperledger/aries-rfcs/tre
 
  RFC Type | RFC/Link to RFC Version | Note
 --- | --- | ---
-Feature | [0592-indy-attachments](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/features/0592-indy-attachments) | Added from AIP V1.0
-Concept | [0441-present-proof-best-practices](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/concepts/0441-present-proof-best-practices) | :new
+Feature | [0592-indy-attachments](https://github.com/hyperledger/aries-rfcs/tree/b982c24b9083dd5dddff6343dbf534cd1cfe36a6/features/0592-indy-attachments) | :new: Evolved from AIP V1.0
+Concept | [0441-present-proof-best-practices](https://github.com/hyperledger/aries-rfcs/tree/b982c24b9083dd5dddff6343dbf534cd1cfe36a6/concepts/0441-present-proof-best-practices) | :new:
 
 #### LDCRED: JSON-LD Based Credentials
 
  RFC Type | RFC/Link to RFC Version | Note
 --- | --- | ---
 Feature | [PR: 0593-json-ld-cred-attach](https://github.com/hyperledger/aries-rfcs/pull/593) | :new:
-Feature | [0510-dif-pres-exch-attach](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/features/0510-dif-pres-exch-attach) | :new
+Feature | [0510-dif-pres-exch-attach](https://github.com/hyperledger/aries-rfcs/tree/b982c24b9083dd5dddff6343dbf534cd1cfe36a6/features/0510-dif-pres-exch-attach) | :new:
 
 #### BBSCRED: BBS+ Based Credentials
 
  RFC Type | RFC/Link to RFC Version | Note
 --- | --- | ---
 Feature | [0xxx-bbs+-signature-credential-exchange-attachments](#) | :new:
-Feature | [0510-dif-pres-exch-attach](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/features/0510-dif-pres-exch-attach) | :new
+Feature | [0510-dif-pres-exch-attach](https://github.com/hyperledger/aries-rfcs/tree/b982c24b9083dd5dddff6343dbf534cd1cfe36a6/features/0510-dif-pres-exch-attach) | :new:
 
 #### DIFPX: DIF Presentation Exchange Support
 
  RFC Type | RFC/Link to RFC Version | Note
 --- | --- | ---
-Feature | [0510-dif-pres-exch-attach](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/features/0510-dif-pres-exch-attach) | :new
+Feature | [0510-dif-pres-exch-attach](https://github.com/hyperledger/aries-rfcs/tree/b982c24b9083dd5dddff6343dbf534cd1cfe36a6/features/0510-dif-pres-exch-attach) | :new:
 
 #### DIDCOMMV2PREP: DIDComm v2 Prep
  RFC Type | RFC/Link to RFC Version | Note
 --- | --- | ---
-Feature | [0334-jwe-envelope](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/features/0334-jwe-envelope) | :question: **Under Consideration**
-Feature | [0587-encryption-envelope-v2](https://github.com/hyperledger/aries-rfcs/tree/9363159d517afa3c17a79b5d8394e933c652de84/features/0587-encryption-envelope-v2) | :new
+Feature | [0587-encryption-envelope-v2](https://github.com/hyperledger/aries-rfcs/tree/b982c24b9083dd5dddff6343dbf534cd1cfe36a6/features/0587-encryption-envelope-v2) | :new:<br>See envelope note below
 
```

Comment:
I'm not sure if this belongs under DIDComm v2 Prep or deserves its own section, but the [formalization of did:peer support in RFC 627](https://github.com/hyperledger/aries-rfcs/tree/master/features/0627-static-peer-dids) may now be worthy of mention?

Reply:
Thanks, Daniel -- I'll get that added.